lix/packages/fs/src/memoryFs.ts

```diff
@@ -164,7 +170,11 @@ export function createNodeishMemoryFs(): NodeishFilesystem {
 
 		_createPlaceholder: async function (
 			path: Parameters<NodeishFilesystem["writeFile"]>[0],
-			options?: Parameters<NodeishFilesystem["writeFile"]>[2]
+			options?: {
```

Comment:
The only place where we are not using options is this test: https://github.com/opral/monorepo/blob/eed92433e5eafe5bd2958a6b356ec480dfb545b8/lix/packages/fs/src/implementations.test.ts#L281 shall we make it non optional?